basic/1-getting-started.spec.ts

```diff
@@ -1,4 +1,5 @@
-import { expect, test } from "@playwright/test";
+import { expect } from "@playwright/test";
+import { test } from "./base.ts";
 
 test("basic test @basic", async ({ page }) => {
   await page.goto("https://todomvc.com/examples/backbone/dist/");
```

Comment:
they changed path - so this one will work
https://todomvc.com/examples/javascript-es6/dist